.github/workflows/build_and_test_check.yml

```diff
@@ -16,6 +16,8 @@ jobs:
           distribution: 'adopt'
 
       - name: Build with Gradle
+        env:
+          ONESIGNAL_API_KEY: ${{ secrets.ONESIGNAL_API_KEY }}
```

Comment:
Jaky je use-case pro to mit produkcni API klic behem testovani? Nevidim v tech testech, ze by se volalo API OneSignalu. Nestacil by tu nejaky fake, pokud to bez te ENV variable spadne?

Reply:
PushNotificationServiceTest - overuju, ze se zpravy spravne odesilaji.